engine/crates/engine/src/registry/resolvers/logged_fetch.rs

```diff
@@ -35,7 +35,7 @@ pub async fn send_logged_request(
                     .headers_mut()
                     .unwrap()
                     .extend(response.headers_mut().drain());
-                let bytes = response.bytes().await?;
+                let bytes = response.bytes().instrument(info_span!("response_data_fetch")).await?;
```

Comment:
Not against this change, but I will be (pleasantly) surprised if it gives us any more timing info

Reply:
Yeah, I also hope it's the backend server we are calling that is slow. But it's good to understand a bit more where the time goes if it's some fucked up WASM/Cloudflare thing we're witnessing here.